doc/pythonreference.rst

```diff
@@ -704,18 +714,18 @@ Classes and Functions
 
    .. warning::
 
-      Environment-objects obtained using the :func:`vpy_current_environment` can directly be used as
+      Environment-objects obtained using the :func:`get_current_environment` can directly be used as
       as a context manager. This can cause undefined behaviour when used in combination with generators and/or
       coroutines.
 
       This context-manager maintains a thread-local environment-stack that is used to restore the previous environment.
       This can cause issues if the frame is suspended inside the block.
 
       A similar problem also existed in previous VapourSynth versions!
-      
+
       .. code::
 
-         env = vpy_current_environment()
+         env = get_current_environment()
          with env:
               yield
```

Comment:
`vpy_current_environment` has this problem. `get_current_environment` cannot be directly used as a context manager.

Reply:
Indeed, sorry!